app/imports/api/methods.js

```diff
@@ -50,15 +50,10 @@ export const saveApplicationSection = new ValidatedMethod({
   run({ type, applicationId, data }) {
 
     if( !this.userId ) {
-      throw new Meteor.Error('Applications.methods.insert.not-logged-in', 'Must be logged in to create an application.');
-    }
-
-    if( this.userId != userId ) {
-      throw new Meteor.Error('Applications.methods.insert.different-user', 'Who you trying to mess with, uh?');
+      throw new Meteor.Error('ApplicationSection.methods.saveSection.not-logged-in', 'Must be logged in to save a section.');
```

Comment:
I removed `this.userId != userId` cuz this method is not receiving a userId. We should actually fix the other method so it uses it's context user id `this.userId` instead of passing the userId from the browser.

http://guide.meteor.com/security.html#user-id-client